### ISSUE TITLE
Shortcut action broken in v1.2.0

### ISSUE THREAD

**Issue by xilopaint** [swiftbar/SwiftBar#123]

**Comment by melonamin** [comment]
Can you give me an example? It works for me

**Comment by xilopaint** [comment]
```
#!/usr/bin/env python3

print('foo | shortcut=CMD+OPT+T')
```

It worked in v1.1.1.

**Comment by melonamin** [comment]
Shortcuts are triggering the action (href, bash or refresh), so this one will open Google:

```
#!/bin/bash
echo "Shortcut Test"
echo "---"
echo "Google| href='https://google.com' shortcut=CMD+OPT+T"
```

What you expect to happen in your example?

**Comment by xilopaint** [comment]
> What you expect to happen in your example?

From the docs:

> Hotkey assigned to item, if item is in header hotkey will show the menu, otherwise hotkey will launch associated action

So the hotkey should show the menu in my example. It worked in v1.1.1.

**Comment by melonamin** [comment]
>From the docs:
>>Hotkey assigned to item, if item is in header hotkey will show the menu, otherwise hotkey will launch associated action

Thanks god we have this docs! :)

I've broken in while implementing #110, now it fixed, will be a part of 1.2.1

**Comment by xilopaint** [comment]
> I've broken in while implementing #110, now it fixed, will be a part of 1.2.1

I will start a campaign in favor of the streaming support. 😄
